.mypy.ini

```diff
@@ -34,3 +34,9 @@ strict_optional = true
 warn_no_return = true
 warn_redundant_casts = true
 warn_unused_ignores = true
+
+[mypy-towncrier.*]
+ignore_missing_imports = true
+
+[mypy-sphinxcontrib.towncrier.*]
+ignore_missing_imports = true
```

Comment:
These were just examples from another project. We don't have these deps currently. So we don't need to ignore them as they're never encountered since the imports don't exist.